### PR TITLE
[StructuralMechanicsApplication] Compute reactions on explicit computations

### PR DESCRIPTION
**Pull request by loumalouomega**
This uses the residual values to update the reactions if needed during explicit computations

### CODE REVIEW

File: applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/mechanical_explicit_strategy.hpp [hunk 1]
```diff
@@ -456,6 +461,62 @@ class MechanicalExplicitStrategy
...
+    void CalculateReactions(
+        typename TSchemeType::Pointer pScheme,
+        ModelPart& rModelPart,
+        TSystemMatrixType& rA,
+        TSystemVectorType& rDx,
+        TSystemVectorType& rb
+        )
+    {
+        // We iterate over the nodes
+        auto r_nodes = rModelPart.Nodes();
```
Comment (philbucher) [inline applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/mechanical_explicit_strategy.hpp]:
```suggestion
        auto& r_nodes = rModelPart.Nodes();
```

**Review by philbucher — COMMENTED** [review]
looks ok from my side

I am not familiar with the algorithm though, that's why @KlausBSautter should approve

@KlausBSautter note that if you don't want the reactions to be computed you should now disable them, they are computed by default in StructuralMechanics

File: applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/mechanical_explicit_strategy.hpp (hunk 1 above)
Comment (loumalouomega) [reply applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/mechanical_explicit_strategy.hpp]: Don

**Review by KlausBSautter — APPROVED** [review]

**Comment by philbucher** [comment]
there is no test yet right?

**Pull request merged**